### DISCUSSION TITLE
Can we hide placeholder fields?

### DISCUSSION THREAD

**Discussion by JoelBeer** [filamentphp/filament#11244] [Help]
### Package

Form builder

### Package Version

v3.1.32

### How can we help you?

Hey, 

I was wondering if it is possible to hide placeholder fields with a form. 

For example. The  'core' value is a bool and is part of the model. 

```           
 Placeholder::make('Core fields cannot be deleted OR made non-required')->hidden(function (Get $get): bool {
                return $get('core'); // returns null 
            }),
```

I can change the content based on this "core" attribute but in the above example `$get('core');` is null but is an actual bool when passing the same closure into the `content()` function see below.

```
   Placeholder::make('Core')->content(function (Get $get): string {
                if ($get('core')) { // Core is true/false
                    return 'Custom core content';
                }
                return '';
            }),
```

I also tried the `hiddenLabel()` approach but the html is still generated and adds unwanted whitespace

Any help would be appreciated.

Joel

**Comment by leandrocfe** [comment]
It should work. Can you share the whole code please?
